src/components/PageTitle.tsx

```diff
@@ -47,34 +47,26 @@ export const PageTitle: React.FC<Props> = ({
     {hasData && !loading && (
       <PageHeader size={size}>
         <TitleWrapper size={size}>
-          <FlexRow>
-            <TitleTypography size={size}>
-              <span data-cy="page-title">
-                {title}
-                {"  "}
-              </span>
-            </TitleTypography>
-            <BadgeWrapper size={size}>{badge}</BadgeWrapper>
-          </FlexRow>
+          <TitleTypography size={size}>
+            <span data-cy="page-title">
+              {title}
+              {"  "}
+              <BadgeWrapper size={size}>{badge}</BadgeWrapper>
+            </span>
```

Comment:
You can remove the `BadgeWrapper` outside of the span to prevent the `{""}` weirdness (it will still work as intended